src/ImagineFormat.jl

```diff
@@ -60,8 +60,9 @@ function load(io::Stream{format"Imagine"}; mode="r")
     pstart = h["piezo"]["stop position"]
     pstop = h["piezo"]["start position"]
     if length(sz)>2
-        dz = abs(pstart - pstop)/(sz[3]-1)
-    else dz = 0.0 end
+        #Setting dz to something > 0.0 prevents an error when creating an AxisArray with a zero step
+        dz = (pstart == pstop) ? 1.0 : abs(pstart - pstop)/(sz[3]-1)
```

Comment:
Sorry I failed to notice this.

Should we also check that `sz[3] == 1`? In such cases would it be better to return a `space × space × time` array?

Reply:
Also see fcda3d2835fc8a59bb62c07dbaf491c96156d588